war/yarn.lock

```diff
@@ -2,31 +2,904 @@
 # yarn lockfile v1
 
 
+"@babel/cli@^7.7.4":
```

Comment:
Is this file necessary? It seems like something automatically generated?

Reply:
This file is autogenerated, so I guess _yarn_ considers it as a needed dependency.

Reply:
It’s a lock file for repeatable builds and should be checked in

Reply:
I'm not sure of understanding. It's autogenerated but it needs to be checked in?